src/sentry/dynamic_sampling/prioritise_transactions.py

```diff
@@ -0,0 +1,304 @@
+import logging
+import time
+from datetime import datetime, timedelta
+from typing import Iterator, List, Optional, Tuple, TypedDict, cast
+
+from snuba_sdk import (
+    AliasedExpression,
+    Column,
+    Condition,
+    Direction,
+    Entity,
+    Function,
+    Granularity,
+    LimitBy,
+    Op,
+    OrderBy,
+    Query,
+    Request,
+)
+
+from sentry.sentry_metrics import indexer
+from sentry.snuba.dataset import Dataset, EntityKey
+from sentry.snuba.metrics.naming_layer.mri import TransactionMRI
+from sentry.snuba.referrer import Referrer
+from sentry.utils.snuba import raw_snql_query
+
+logger = logging.getLogger(__name__)
+MAX_SECONDS = 60
+CHUNK_SIZE = 1000
```

Comment:
do we want to increase `CHUNK_SIZE` to 10000 (Snuba's limit)? or did we turn down that idea for some reason?

https://www.notion.so/sentry/Project-bias-analysis-44a0c93358c94979b18ebd06a6b7f52e?pvs=4#34a2cff938f04d34a4b62a1c75acaadd

Reply:
Yes thanks, I missed that.